archetypes/article/index.md

```diff
@@ -0,0 +1,9 @@
+---
+date: {{ .Date }}
+author: someone@example.com
+title: '{{ replace .File.ContentBaseName `-` ` ` | title }}'
+lastmod: :git
+draft: true
```

Comment:
Do we want date?
This is what we have been using for `lastmod` in the config of each site:
```yaml
 lastmod:
    - lastmod
    - :fileModTime
    - :default
```
maybe should just be added to the config rather than ever file's frontmatter for consistency?